archive/development/22-8-1-dev-fix/development.json

```diff
@@ -0,0 +1,551 @@
+{
```

Comment:
also commit the verification inputs which should be output as part of the deploy script whenever new contracts are deployed. those can be used to verify the contracts on etherscan

Reply:
Will recommit that with new deployment instances